adafruit_si1145.py

```diff
@@ -38,10 +38,15 @@
 SI1145_DEFAULT_ADDRESS = const(0x60)
 SI1145_PART_ID = const(0x00)
 SI1145_HW_KEY = const(0x07)
+_COEFF_0 = const(0x13)
+_COEFF_1 = const(0x14)
+_COEFF_2 = const(0x15)
+_COEFF_3 = const(0x16)
 SI1145_PARAM_WR = const(0x17)
 SI1145_COMMAND = const(0x18)
 SI1145_RESPONSE = const(0x20)
 SI1145_ALS_VIS_DATA0 = const(0x22)
+SI1145_UV_INDEX_DATA0 = const(0x2C)
 SI1145_PARAM_RD = const(0x2E)
```

Comment:
We chatted about this in the meeting today. Basically, you *should* use register if its model matches the device's registers.

You can save space in this library by reducing these variables to, for example, `_UV_INDEX_DATA0 = const(0x2c)`. For `const` to drop a variable name it must start with `_`, otherwise the whole variable name is kept around in case code outside this module references it. (`_` prefix in Python implies that something should be private.) You can also drop the `SI1145` bit because it duplicates the context of the file/module name that already includes `si1145`.

Reply:
@tekktrik Agree with above. OK to use register and please add the `_`. I just forgot about that being a requirement. Also the degree to which the const names need to be unique, i.e. having `SI1145` in all of them.